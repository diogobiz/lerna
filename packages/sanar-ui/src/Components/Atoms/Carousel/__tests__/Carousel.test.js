import React from 'react'
import TestRenderer from 'react-test-renderer'

import ESCarousel from '../'

it('renders correctly', () => {
    const component = (
        <ESCarousel arrows={true}>
            {[0, 1, 2, 3, 4, 6, 7].map((e, i) => (
                <div key={i}>
                    <h2>{i}</h2>
                </div>
            ))}
        </ESCarousel>
    )

    const tree = TestRenderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
})
