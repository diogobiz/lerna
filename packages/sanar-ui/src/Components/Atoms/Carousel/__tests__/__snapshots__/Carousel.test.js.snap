// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="es-carousel"
>
  <div
    className="ant-carousel"
  >
    <div
      className="slick-slider slick-initialized"
      dir="ltr"
    >
      <button
        className="es-button slick-arrow slick-prev es-button__xsmall circle ant-btn"
        data-testid="es-button"
        onClick={[Function]}
        type="button"
      >
        <i
          className="anticon es-eva-icon es-eva-icon--medium"
        >
          <i
            data-eva="arrow-ios-back-outline"
            data-eva-fill="#fff"
            data-eva-height="18px"
            data-eva-width="18px"
          />
        </i>
      </button>
      <div
        className="slick-list"
        onClick={[Function]}
        onKeyDown={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={null}
        onMouseMove={null}
        onMouseUp={[Function]}
        onTouchCancel={null}
        onTouchEnd={[Function]}
        onTouchMove={null}
        onTouchStart={[Function]}
        style={Object {}}
      >
        <div
          className="slick-track"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onMouseOver={[Function]}
          style={
            Object {
              "WebkitTransform": "translate3d(0px, 0px, 0px)",
              "WebkitTransition": "",
              "msTransform": "translateX(0px)",
              "opacity": 1,
              "transform": "translate3d(0px, 0px, 0px)",
              "transition": "",
            }
          }
        >
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={-1}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  6
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={false}
            className="slick-slide slick-active slick-current"
            data-index={0}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  0
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={1}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  1
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={2}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  2
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={3}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  3
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={4}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  4
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={5}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  5
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide"
            data-index={6}
            onClick={[Function]}
            style={
              Object {
                "outline": "none",
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  6
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={7}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  0
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={8}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  1
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={9}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  2
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={10}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  3
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={11}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  4
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={12}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  5
                </h2>
              </div>
            </div>
          </div>
          <div
            aria-hidden={true}
            className="slick-slide slick-cloned"
            data-index={13}
            onClick={[Function]}
            style={
              Object {
                "width": 0,
              }
            }
            tabIndex="-1"
          >
            <div>
              <div
                style={
                  Object {
                    "display": "inline-block",
                    "width": "100%",
                  }
                }
                tabIndex={-1}
              >
                <h2>
                  6
                </h2>
              </div>
            </div>
          </div>
        </div>
      </div>
      <button
        className="es-button slick-arrow slick-next es-button__xsmall circle ant-btn"
        data-testid="es-button"
        onClick={[Function]}
        type="button"
      >
        <i
          className="anticon es-eva-icon es-eva-icon--medium"
        >
          <i
            data-eva="arrow-ios-forward-outline"
            data-eva-fill="#fff"
            data-eva-height="18px"
            data-eva-width="18px"
          />
        </i>
      </button>
      <ul
        className="slick-dots slick-dots-bottom"
        onMouseEnter={null}
        onMouseLeave={null}
        onMouseOver={null}
        style={
          Object {
            "display": "block",
          }
        }
      >
        <li
          className="slick-active"
        >
          <button
            onClick={[Function]}
          >
            1
          </button>
        </li>
        <li
          className=""
        >
          <button
            onClick={[Function]}
          >
            2
          </button>
        </li>
        <li
          className=""
        >
          <button
            onClick={[Function]}
          >
            3
          </button>
        </li>
        <li
          className=""
        >
          <button
            onClick={[Function]}
          >
            4
          </button>
        </li>
        <li
          className=""
        >
          <button
            onClick={[Function]}
          >
            5
          </button>
        </li>
        <li
          className=""
        >
          <button
            onClick={[Function]}
          >
            6
          </button>
        </li>
        <li
          className=""
        >
          <button
            onClick={[Function]}
          >
            7
          </button>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
